### PR TITLE
feat: Check for latest version of Uno.Check

### PR DESCRIPTION
**Pull request by MartinZikmund**

### CODE REVIEW

**Comment by MartinZikmund** [comment]
@jeromelaban Noticed now there is a "required version" check in `ToolInfo.Validate` which checks against required version from manifest, but maybe this approach is better as it will always point to the latest version, so it could replace/complement the existing one, right?

File: UnoCheck/CheckCommand.cs [hunk 1]
```diff
@@ -1,327 +1,355 @@
...
-			AnsiConsole.MarkupLine("This tool will check your Uno Platform development environment.");
-			AnsiConsole.MarkupLine("If problems are detected, it will offer the option to try and fix them for you, or suggest a way to fix them yourself.");
-			AnsiConsole.Render(new Rule());
+                AnsiConsole.Render(new Rule());               
+            }
 
-			if (!Util.IsAdmin() && Util.IsWindows)
-			{
-				var suTxt = Util.IsWindows ? "Administrator" : "Superuser (su)";
+            if (latestVersion is null || currentVersion < latestVersion)
```
Comment (jeromelaban) [inline UnoCheck/CheckCommand.cs:60]: We're likely need to skip this if we're specifying a manifest on the CLI and/or that we're running with the `--ci` flag.

**Comment by jeromelaban** [comment]
> @jeromelaban Noticed now there is a "required version" check in `ToolInfo.Validate` which checks against required version from manifest, but maybe this approach is better as it will always point to the latest version, so it could replace/complement the existing one, right?

Required version is useful when manifests are explicitly provided, but that's not something end-users will use. Your approach addresses another need, as it pokes at the latest on the feed. Thanks!

File: UnoCheck/CheckCommand.cs (hunk 1 above)
Comment (MartinZikmund) [reply UnoCheck/CheckCommand.cs]: Done!

**Review by jeromelaban — APPROVED** [review]

**Pull request merged**